### PR TITLE
EDU-663 update and restyle abc notation

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-663

### CODE REVIEW

File: src/components/media-player-controls.js [hunk 1]
```diff
@@ -82,6 +78,12 @@ function MediaPlayerControls({
     return <Menu items={items} onClick={handleSettingsMenuItemClick} />;
   };
 
+  const renderTimeDisplay = () => {
+    return durationInMilliseconds
+      ? <Fragment>{formatMillisecondsAsDuration(playedMilliseconds)}&nbsp;/&nbsp;{formatMillisecondsAsDuration(durationInMilliseconds)}</Fragment>
+      : <Fragment>--:--&nbsp;/&nbsp;--:--</Fragment>;
```
Comment (adrianaluchian) [inline src/components/media-player-controls.js:84]: Nice placeholder idea.

**Comment by adrianaluchian** [comment]
Seems now we get an exception when trying to download anything from internal storage uploaded in video, audio, etc. plugins 🤔

**Comment by ahelmberger** [comment]
> Seems now we get an exception when trying to download anything from internal storage uploaded in video, audio, etc. plugins thinking

Yeah, this is also happening on the main branch (but only on `localhost`). It's a CORS problem. I know the axios request is there as a workaround in order to always download instead of opening a new browser tab. But we would save ourself a some trouble if we'd just let the browser handle the download naturally (by just setting the `href` to the actual URL). Maybe we can discuss in the near future.

File: src/components/media-player-controls.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/media-player-controls.js:84]: Thanks, I thought we should indicate somehow that the file is not "loaded" yet (and I found "0:00" total length confusing) :-)

**Comment by adrianaluchian** [comment]
> > Seems now we get an exception when trying to download anything from internal storage uploaded in video, audio, etc. plugins thinking
> 
> Yeah, this is also happening on the main branch (but only on `localhost`). It's a CORS problem. I know the axios request is there as a workaround in order to always download instead of opening a new browser tab. But we would save ourself a some trouble if we'd just let the browser handle the download naturally (by just setting the `href` to the actual URL). Maybe we can discuss in the near future.

Oh yes, of course, it's CORS. Ok, we can live with that and we can always use the tunnel to test such cases.

**Review by adrianaluchian — APPROVED** [review]
Looks good and unified now 👏

**Pull request merged**